### PR TITLE
Added HW 1,2,3

### PR DESCRIPTION
**Pull request by djvvvm**

### CODE REVIEW

File: dmisia/HW1/Practical_Task_1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name = input('What is your name? ')
```
Comment (kolyasalubov) [inline dmisia/HW1/Practical_Task_1.py:1]: good job

File: dmisia/HW1/Practical_Task_2.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+a = int(input('Write "a" number:'))
```
Comment (kolyasalubov) [inline dmisia/HW1/Practical_Task_2.py:1]: good job

File: dmisia/HW3/Practical_Task_1.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+48.Unless explicitly silenced.
+49.In the face of ambiguity, refuse the temptation to guess.
+50.There should be one-- and preferably only one --obvious way to do it.
+51.Although that way may not be obvious at first unless you're Dutch.
+52.Now is better than never.
+53.Although never is often better than *right* now.
+54.If the implementation is hard to explain, it's a bad idea.
+55.If the implementation is easy to explain, it may be a good idea.
+56.Namespaces are one honking great idea -- let's do more of those!"""
+lines = str1.splitlines()
```
Comment (kolyasalubov) [inline dmisia/HW3/Practical_Task_1.py:58]: good job

File: dmisia/HW3/Practical_Task_2.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+num = int(input('Enter four-digit natural number: '))
```
Comment (kolyasalubov) [inline dmisia/HW3/Practical_Task_2.py:1]: good job

please look at this implementation:

number = int(input("Enter your number: "))

one_div, remainder = divmod(number, 1000)
two_div, remainder = divmod(remainder, 100)
three_div, four_div = divmod(remainder, 10)

Item 1: find of product of the digits of this number:
prod_of_dig = one_div * two_div * three_div * four_div
print("Product of the digits of this number:", prod_of_dig)

Item 2: write the number in reverse order:
str_number = str(number)[::-1]
print("Reverse of our number:", str_number)

Item 3: in ascending order, you need to sort the number included in the given number:
sort_number = ''.join(sorted(str(number)))
print("Ascending order of digits in the number:", sort_number)

File: dmisia/HW3/Practical_Task_3.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+a = int(input('Enter first digit: '))
```
Comment (kolyasalubov) [inline dmisia/HW3/Practical_Task_3.py:1]: please look at this 
a, b = b, a

File: dmisia/HW3/Practical_Task_2.py (hunk 4 above)
Comment (djvvvm) [reply dmisia/HW3/Practical_Task_2.py:1]: thanks, it is much simpler

File: dmisia/HW4/Practical_Task_1.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+temp1 = int(input('Enter the temperature in Celsius: '))
```
Comment (kolyasalubov) [inline dmisia/HW4/Practical_Task_1.py:1]: good job

File: dmisia/HW5/Practical_Task_1.py [hunk 7]
```diff
@@ -0,0 +1,7 @@
+list1 = [1, 287, 82, 47, 22]
```
Comment (kolyasalubov) [inline dmisia/HW5/Practical_Task_1.py:1]: good job

File: dmisia/HW5/Practical_Task_2.py [hunk 8]
```diff
@@ -0,0 +1,7 @@
+n = int(input("Enter your stop number: "))
```
Comment (kolyasalubov) [inline dmisia/HW5/Practical_Task_2.py:1]: good job

File: dmisia/HW5/Practical_Task_3.py [hunk 9]
```diff
@@ -0,0 +1,8 @@
+fct = int(input("Enter your number: "))
```
Comment (kolyasalubov) [inline dmisia/HW5/Practical_Task_3.py:1]: good job

File: dmisia/HW6/Practical_Task_2.py [hunk 10]
```diff
@@ -0,0 +1,5 @@
+login = input("Enter your login: ")
```
Comment (kolyasalubov) [inline dmisia/HW6/Practical_Task_2.py:1]: good job

File: dmisia/HW6/Practical_Task_1.py [hunk 11]
```diff
@@ -0,0 +1,7 @@
+num2 = [i for i in range(1,11) if i % 2 == 0]
```
Comment (kolyasalubov) [inline dmisia/HW6/Practical_Task_1.py:1]: good job

**Pull request merged**